doc/constrain_R14_trend.md

```diff
@@ -0,0 +1,88 @@
+## Contrain neutron stars Equation of State (EoS) 
+
```

Comment:
I would put the link to Zenodo and Tim's EOS repository and the discussion of which files to get up here.

Reply:
I recall a few were decided to not be fully converged. Clarify with Peter

Reply:
ok